robot/EDA/resources/locators.py

```diff
@@ -159,13 +159,13 @@
     },
     "account_types": {
         "administrative": "//span[contains(text(),'Administrative')]/parent::*",
-        "academic": "//div[contains(@class,'slds-form-element__control')]//span[contains(text(),'Academic Program')]",
-        "administrativecheckbox": "//div[contains(@class,'slds-form-element__control')]//span[contains(text(),'Administrative')]",
-        "business": "//div[contains(@class,'slds-form-element__control')]//span[contains(text(),'Business Organization')]",
-        "educational": "//div[contains(@class,'slds-form-element__control')]//span[contains(text(),'Educational Institution')]",
-        "household": "//div[contains(@class,'slds-form-element__control')]//span[contains(text(),'Household Account')]",
-        "sports": "//div[contains(@class,'slds-form-element__control')]//span[contains(text(),'Sports Organization')]",
-        "university": "//div[contains(@class,'slds-form-element__control')]//span[contains(text(),'University Department')]",
+        # "academic": "//div[contains(@class,'slds-form-element__control')]//span[contains(text(),'Academic Program')]",
```

Comment:
If we're not going to use these old locators, you might was well delete them rather than just comment them out. If we decide we need them again, we can find them in the git history of the file.

Reply:
Yes! I just commented them out for testing, in case my code didn't work. Now that it does, I'll remove.